scripts/gencert.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=Generate xapi ssl certificate
-Requires=xapi.service forkexecd.service xapi-init-complete.target
-After=xapi.service forkexecd.service xapi-init-complete.target
+Requires=xapi.service forkexecd.service
+After=xapi.service forkexecd.service
```

Comment:
Can you remove xapi.service too? All you need there is the inventory, do you actually need to communicate with XAPI?

Reply:
you're right